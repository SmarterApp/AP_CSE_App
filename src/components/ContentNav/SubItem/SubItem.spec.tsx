import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { SubItem } from '.';

describe('SubItem', () => {
  it('should be active', () => {
    const subItemActive = mount(<SubItem name={'Test Name'} active={true} />);

    subItemActive.simulate('click');
    expect(toJson(subItemActive)).toMatchSnapshot();

    subItemActive.unmount();
  });
});
