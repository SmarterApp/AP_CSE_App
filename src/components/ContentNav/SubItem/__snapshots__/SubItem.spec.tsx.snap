// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubItem should be active 1`] = `
<Component
  active={true}
  name="Test Name"
>
  <li
    className="active"
    onClick={[Function]}
    role="menuitem"
  >
    <p>
      Test Name
    </p>
    <style
      jsx={true}
    >
      
        * {
          font-family: PT Sans Caption;
        }
        li {
          box-sizing: border-box;
          border-left: 3px solid #ffffff;
          display: flex;
          align-items: center;
          color: #006298;
          font-size: 16px;
          font-weight: normal;
          text-indent: 1em;
          height: 3em;
        }
        li :first-child {
          border-top: 1px solid #000;
        }
        p {
          margin-right: 2em;
        }
        .active {
          box-sizing: border-box;
          border-left: 3px solid #006298;
          font-weight: bold;
        }
        :not(.active) {
          background-color: #ffffff;
        }
      
    </style>
  </li>
</Component>
`;
